tools/test-js-with-puppeteer

```diff
@@ -118,7 +119,14 @@ def run_tests(files: Iterable[str], external_host: str) -> None:
                     response = input('Tests failed. Press Enter to re-run tests, "q" to quit: ')
         else:
             ret = 1
-            ret, current_test_num = run_tests()
+            for loop_num in range(loop):
+                print(f"\n\nRunning tests in loop {loop_num}/{loop}\n")
```

Comment:
Is the wording OK here? Any other ideas? (we'll be printing it even if `--loop` isn't passed. Made it this way just to keep code simple.)

Reply:
The wording seems fine here, but don't you want `loop_num + 1`?

Reply:
Oh, that's correct. Fixed it, thanks!